include/zephyr/net/wifi_mgmt.h

```diff
@@ -43,7 +43,8 @@ enum net_request_wifi_cmd {
 	NET_REQUEST_WIFI_CMD_PS_CONFIG,
 	NET_REQUEST_WIFI_CMD_REG_DOMAIN,
 	NET_REQUEST_WIFI_CMD_PS_TIMEOUT,
-	NET_REQUEST_WIFI_CMD_MAX
+	NET_REQUEST_WIFI_CMD_MAX,
+	NET_REQUEST_WIFI_CMD_LISTEN_INTERVAL,
```

Comment:
NET_REQUEST_WIFI_CMD_LISTEN_INTERVAL should come after NET_REQUEST_WIFI_CMD_MAX. I see that this has been corrected in the next commit but please correct it in this commit as well

Reply:
> NET_REQUEST_WIFI_CMD_LISTEN_INTERVAL should come after NET_REQUEST_WIFI_CMD_MAX. I see that this has been corrected in the next commit but please correct it in this commit as well

correction:  NET_REQUEST_WIFI_CMD_LISTEN_INTERVAL should come before NET_REQUEST_WIFI_CMD_MAX.